source/release-notes.txt

```diff
@@ -10,6 +10,78 @@ Release Notes
    :depth: 1
    :class: twocols
 
+
+|compass| 1.35.0
+----------------
+
+*Released January 11, 2023*
+
+New Features
+
+- Update export modal to LeafyGreen components (:issue:`COMPASS-6220`)
+- Replace types dropdown with LG select
+- Use leafygreen combobox to select stages
+- Replace export-to-language with leafygreen components (:issue:`COMPASS-6219`)
+- Add connection import/export UI
+- Convert compass query history to new components (:issue:`COMPASS-6221`)
+- Use the same date hook in query history as in saved aggregations (:issue:`COMPASS-6221`)
+- Add forceConnectionOptions option (:issue:`COMPASS-6068`)
+- Implement readOnly option (:issue:`COMPASS-6064`)
+- Update import modal to LeafyGreen components (:issue:`COMPASS-6220`)
+- Add --username and --password for auto-connect (:issue:`COMPASS-6216`)
+- Expose protectConnectionStrings in settings UI (:issue:`COMPASS-6262`)
+- Kerberos password field setting (:issue:`COMPASS-5950`)
+- Add maxTimeMS as setting (:issue:`COMPASS-6063`)
+- Update compass validation components to leafygreen (:issue:`COMPASS-6237`)
+- Update explain plan components (:issue:`COMPASS-6236`)
+- Implement enableDevTools option (:issue:`COMPASS-6061`), (:issue:`COMPASS-5615`)
+- Use rebranded components in the document table view
+- Add tracking event when stage value changes (:issue:`COMPASS-6310`)
+- Update Compass aggregations modals (:issue:`COMPASS-6286`)
+- Add LG darkTheme support for table view
+- Remove trackErrors setting (:issue:`COMPASS-5708`)
+- Move all autoupdates logic to compass main process, allow to dismiss updates (:issue:`COMPASS-6057`) (:issue:`COMPASS-6303`)
+- Convert more insert dialog code to compass components & leafygreen (:issue:`COMPASS-6285`)
+- Register Compass as a protocol handler for ``mongodb://`` (:issue:`COMPASS-6085`)
+- Add ``--show-example-config`` flag (:issue:`COMPASS-6084`)
+- Cancellable aggregate and schema analysis (:issue:`COMPASS-5668`)
+- Cancellable find and explain (:issue:`COMPASS-5668`)
+- Implement new input docs card design (:issue:`COMPASS-6234`)
+- Update scrollbar styles (:issue:`COMPASS-5597`)
+- Cancellable counts (:issue:`COMPASS-5668`)
+- Update aggregations stage components (:issue:`COMPASS-6234`)
+- Enable pipeline as text feature (:issue:`COMPASS-6299`)
+- Index tab UI improvements (:issue:`COMPASS-6323`), (:issue:`COMPASS-6329`)
+- Add refresh document count in aggregation results (:issue:`COMPASS-6156`)
+- Confirm when deleting pipeline (:issue:`COMPASS-4137`)
+
+Bug Fixes
```

Comment:
[suggestion] Should this be "Bug Fixes:"?

Reply:
Updated.